### DISCUSSION TITLE
Static files with 2 dots are not found

### DISCUSSION THREAD

**Discussion by ecmel** [encode/starlette#1547] [Potential Issue] [answered]
Hello, 

I am using Parcel for client bundling. The generated files are like:

```
main.21ee2e74.css
main.d37327fa.js
main.html
```
I am serving the files using StaticFiles.

I get main.html with no problem but first and second files give me 404 not found error.

I suspect it is related with 2 dots in the filename.

**Comment by Kludex** [comment]
Works just fine here. Can you create a minimal example in as a GitHub repository or gist, so we can reproduce?

**Comment by Kludex** [comment, marked as answer]
Closing as I couldn't reproduce it.
